benchopt/plotting/plot_objective_curve.py

```diff
@@ -34,11 +33,10 @@ def plot_objective_curve(df, plotly=False, suboptimality=False,
         The rendered figure, used to create HTML reports.
     """
     if plotly:
-        from plotly.validators.scatter.marker import SymbolValidator
-        markers = SymbolValidator().values[::12]
-        print(markers)
+        MARKERS = {i: i for i, v in enumerate(plt.Line2D.markers)}
     else:
-        markers = list(Line2D.markers)
+        MARKERS = {i: v for i, v in enumerate(plt.Line2D.markers)}
```

Comment:
If this is not a constant, use lowercase.
```suggestion
        markers = {i: v for i, v in enumerate(plt.Line2D.markers)}
```

Reply:
actually you are the one who got from markers to MARKERS in the first place?

CMAP has the same issue btw;)

Reply:
CMAP is a global constant of the module -> uppercase 
I put MARKER as a global constant to but you revert it to local so it is lowercase.

Sorry for my TOC 😅